src/modules/Dashboard/features/Applets/Applets.tsx

```diff
@@ -73,46 +98,152 @@ export const Applets = () => {
     }
   };
 
+  const onCloseCallback = () => {
+    fetchData();
+  };
+
+  const fetchData = async (getAppletsParams?: GetAppletsParams) => {
```

Comment:
Could you pls move code for example to the Applets.hooks.ts to avoid large amount of code here?